### ISSUE TITLE
Add support for PropertyLayoutVariant in @codotype/ui

### ISSUE THREAD

**Issue by aeksco** [codotype/codotype#159]
I'll also need to build out the `PropertyLayoutVariant` enum in `@codotype/types`. Include:

- `PropertyLayoutVariant.basic`
- `PropertyLayoutVariant.card`
- `PropertyLayoutVariant.detail`
- `PropertyLayoutVariant.detailCard`

**Comment by aeksco** [comment]
Updated this to include the following:
```ts
export declare enum PropertyLayoutVariant {
    COL_3 = "COL_3",
    COL_4 = "COL_4",
    COL_6 = "COL_6",
    COL_8 = "COL_8",
    COL_12 = "COL_12",
    CARD_COL_3 = "CARD_COL_3",
    CARD_COL_4 = "CARD_COL_4",
    CARD_COL_6 = "CARD_COL_6",
    CARD_COL_8 = "CARD_COL_8",
    CARD_COL_12 = "CARD_COL_12"
}
```

**Comment by aeksco** [comment]
Resolved in #185
